utility/firebase.js

```diff
@@ -455,3 +455,72 @@ export const subscribeToLivesiteData = (callback) => {
 export const updateLivesiteData = async (data) => {
   return livesiteDocRef.update(data);
 };
+
+export const getLivesiteEvent = (eventID, data) => {
+  return data
+    ? {
+        ...data,
+        eventID,
+        key: data.key || eventID,
+        name: data.name || 'Empty event field',
+        description: data.description || 'Empty text description for event',
+        lastModified: data.lastModified
+          ? formatDate(data.lastModified.seconds)
+          : formatDate(getTimestamp().seconds),
+        lastModifiedBy: data.lastModifiedBy || 'Unknown user',
+      }
```

Comment:
I think `startTime` and `endTime` should be formatted here - I recently refactored the `formatDate` helper so it should work now